api/src/main/java/org/pmiops/workbench/auth/ProfileService.java

```diff
@@ -72,6 +72,7 @@ public Profile getProfile(User user) {
     profile.setAreaOfResearch(user.getAreaOfResearch());
     profile.setRequestedIdVerification(user.getRequestedIdVerification());
     profile.setTwoFactorEnabled(user.getTwoFactorEnabled());
+    profile.setDisabled(user.getDisabled());
```

Comment:
(SIde note) Ahh, the fact that we have to do stuff like this for the Profile data model still bugs me. It's a code smell that maybe we're not structuring things right. Perhaps the Profile object should contain a full User object inside of it, rather than a shallow copy of all of these fields...

Cleanup for a rainy day sometime.

Reply:
Agreed. I really hate that we do all this copying. And it led to a problem where we did have a disabled field on profile, but it wasn't actually being copied so I was just confused. 👍 to fixing it someday.

Reply:
Follow up ticket here: 
https://precisionmedicineinitiative.atlassian.net/secure/RapidBoard.jspa?rapidView=30&projectKey=RW&view=planning&selectedIssue=RW-2132&epics=visible